.buildkite/pipeline.yml

```
@@ -3,6 +3,10 @@
 env:
   VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
   DOCKER_REGISTRY: "docker.elastic.co"
+  ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true' 
```

I think it should be without the prefix ([example](https://github.com/elastic/buildkite-shipit/blob/0ce7bc8e6470dc2411c357428fc4a0c6f57042c0/.buildkite/pipeline.yml#L3))
```suggestion
  SLACK_NOTIFICATIONS_ENABLED: 'true' 
```